src/components/LoginButton/LoginButton.tsx

```diff
@@ -1,9 +1,21 @@
+"use client";
+import React, { useEffect, useState } from "react";
 import { useAccount } from "wagmi";
-import DisconnectButton from "./DisconnectButton";
 import { ConnectAccount } from "@coinbase/onchainkit/wallet";
 
+import DisconnectButton from "./DisconnectButton";
+
 const LoginButton = () => {
   const { address } = useAccount();
+  const [isMounted, setIsMounted] = useState(false);
+
+  useEffect(() => {
+    setIsMounted(true);
+  }, []);
+
+  if (!isMounted) {
+    return null;
+  }
```

Comment:
why are changes to `<LoginButton />` required for implementation of `<ProjectPage />`?